great_expectations/compatibility/aws.py

```diff
@@ -38,3 +41,18 @@
     from sqlalchemy_redshift import dialect as redshiftdialect
 except (ImportError, AttributeError):
     redshiftdialect = REDSHIFT_NOT_IMPORTED
+
+try:
+    import pyathena
+except ImportError:
+    pyathena = ATHENA_NOT_IMPORTED  # type: ignore[assignment]
+
+try:
+    from pyathena import sqlalchemy_athena
```

Comment:
nit: 

for some uniformity in our imports can this be `anthenasqlalchemy`

Reply:
@TrangPham I had to break this pattern, because the actual `pyathena` has the type `sqlalchemy_athena` -- and I did not want to "correct the source" when it is "already so close".  Thanks.